lib/php/src/ext/thrift_protocol/php_thrift_protocol.cpp

```diff
@@ -537,6 +537,9 @@ void binary_deserialize(int8_t thrift_typeID, PHPInputTransport& transport, zval
       }
 
       zval* spec = zend_read_static_property(Z_OBJCE_P(return_value), "_TSPEC", sizeof("_TSPEC")-1, false);
+      if (spec) {
```

Comment:
After `zend_read_static_property` with `slient=false`, we should detect exception.
If no exception, we needn't detect spec is null.

but we need detect that `Z_TYPE_P(prop) == IS_REFERENCE`

Reply:
ZVAL_DEREF checks is reference:
https://github.com/php/php-src/blob/ffc734b2a455e2f2748f18a54ab597f7e0c715ba/Zend/zend_types.h#L944

I removed if statement